recipes/r-igraph/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = '1.0.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-igraph
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: igraph_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/igraph_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/igraph/igraph_{{ version }}.tar.gz
+  sha256: dc64ed09b8b5f8d66ed4936cde3491974d6bc5178dd259b6eab7ef3936aa5602
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-matrix
+    - r-nmf
+    - r-irlba
+    - r-magrittr
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
+    - r-matrix
+    - r-nmf
+    - r-irlba
+    - r-magrittr
+    - libgcc  # [not win]
```

Comment:
You need `gmp` for build and run. (For build, `gmp` is pulled in by the `gcc` package)